src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -90,10 +102,13 @@ internal override ContractTask InitializeAsync(ApplicationEngine engine, Hardfor
                 engine.SnapshotCache.Add(_execFeeFactor, new StorageItem(DefaultExecFeeFactor));
                 engine.SnapshotCache.Add(_storagePrice, new StorageItem(DefaultStoragePrice));
             }
+
             if (hardfork == Hardfork.HF_Echidna)
             {
+                engine.SnapshotCache.Add(_blockGenTime, new StorageItem(DefaultBlockGenTime));
```

Comment:
Not appropriate. FS networks are already faster than this. We also need to be able to specify this value in the genesis configuration, so if it's there the value should be taken from config. Similar to https://github.com/nspcc-dev/neo-go/blob/master/docs/node-configuration.md#Genesis-Configuration

Reply:
this is policy contract, it can not read from the config.

Reply:
See https://github.com/nspcc-dev/neo-go/pull/3835 for the reference, please. In particular, we need this:
https://github.com/nspcc-dev/neo-go/blob/e615a90c24c761e3f176b4fa87aad0d14662c37e/pkg/core/native/policy.go#L214-L215